requirements.txt

```diff
@@ -1 +1,2 @@
 pyglet~=1.3
+git+git://github.com/jorgecarleitao/pyglet-gui@master
```

Comment:
Nice!

Reply:
It was added by you on my computer, so indeed Nice! but not my work 😄